gamefixes-steam/368500.py

```diff
@@ -0,0 +1,8 @@
+"""Game fix for Assassin's Creed: Syndicate"""
+
+from protonfixes import util
+
+
+def main() -> None:
+	"""Game ships with outdated and not working uPlay launcher."""
+	util.protontricks('ubisoftconnect')
```

Comment:
Thanks for this.

Also, following the upstream report for this title, I believe another environment seems to also be needed? Namely, PROTON_DISABLE_NVAPI=1

See https://github.com/ValveSoftware/Proton/issues/2464

Reply:
Going by the second to last comment, it looks like there may already be plans to add that to official Proton.

Reply:
We can remove that specific fix once those plans ever come to fruition. For environment variables, there's generally no harm setting it twice. There's only harm when the env var isn't needed anymore and a proper upstream solution has been implemented to resolve the issue. In those cases, we need to pay attention.